### ISSUE TITLE
sos exits with BrokenPipeError

### ISSUE THREAD

**Issue by BoPeng** [vatlab/SOS#490]
Something like

```
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
BrokenPipeError: [Errno 32] Broken pipe
```

need to figure out what is going on.

**Comment by BoPeng** [comment]
Seems to be a common warning (http://stackoverflow.com/questions/16314321/suppressing-printout-of-exception-ignored-message-in-python-3) but not sure how to suppress it in sos.

**Comment by BoPeng** [comment]
Seem to be fixed by #493.
